modules/mo_pytorch/mo_extensions/load/pytorch/hooks.py

```diff
@@ -96,11 +96,73 @@ class Add(nn.Module):
         return forward_hook(Add(), (self, a), output)
 
     def __add__(self, a):
-        res = self._value + a._value
-        class Add(nn.Module):
-            pass
-        return forward_hook(Add(), (self, a), res)
-
+        if isinstance(a, OpenVINOTensor):
+            class Add(nn.Module):
+                pass
+            res = self._value + a._value
+            return forward_hook(Add(), (self, a), res)
+
+        elif isinstance(a, float):
+            class Add(nn.Module):
+                def __init__(self, value):
+                    super().__init__()
+                    self.register_buffer('add', torch.tensor(value))
+
+            res = self._value + a
+            return forward_hook(Add(a), (self,), res)
+
+    def __getitem__(self, n):
```

Comment:
Please rename `n` to something meaningful. In example, `key` (see https://docs.python.org/3/reference/datamodel.html#object.__getitem__)